### ISSUE TITLE
Signup process needs work on mobile

### ISSUE THREAD

**Issue by 75th** [keybase/keybase-issues#579]
I got my acceptance email while I was out at dinner, so I raced on my phone to claim my username. I got signed up, but there were some bumps:
- I used iCloud Keychain to generate a password. It populated the two fields, but didn't trigger whatever event the Javascript was looking at to un-gray-out the Join button, so I had to tab over to the Confirm field, hit the spacebar, and hit backspace.
- Tabbing between the fields was buggy. Can't be much more specific here; either there was serious lag, or iOS 7's tab arrows were sending the cursor to invisible elements somewhere, or something. The page wasn't very cooperative in scrolling the content to wherever the cursor was, either; getting to where the field I was typing into wasn't hidden behind the keyboard was difficult. Sorry I can't be more specific.
- When I finally hit the Join button, the site freaked out a bit and produced the following screenshot:

![img_1495](https://cloud.githubusercontent.com/assets/1731790/2688606/121d8fb2-c2b4-11e3-930b-964fdae33fb1.png)

**TL;DR**: The signup process at some point needs some more testing on phones.

(P.S. — I changed my password later when I was on my laptop, so it may or may not have as many characters as you see in the screenshot.)
